apps/web/src/common/components/Navbar/components/NavbarProfile/index.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @next/next/no-img-element */
+import { Typography } from '@mui/material'
+import React from 'react'
+
+import { ProfileContainer } from './styled'
+
+const NavbarProfile = () => {
+  return (
+    <ProfileContainer>
+      <img
+        src="https://genshin.honeyhunterworld.com/img/ayaka_002.webp"
+        alt="profile image"
+        style={{
+          width: '40px',
+          height: '40px',
+          objectFit: 'cover',
+          borderRadius: '50%',
+        }}
+      />
```

Comment:
Next image ?

Reply:
will do this when fetch user data from API